### PR TITLE
Improve error message when cocotb-config is not found

### PR DESCRIPTION
**Pull request by themperek**
Improve error message when cocotb-config is not found when in compatibility mode

### CODE REVIEW

File: makefiles/Makefile.inc [hunk 1]
```diff
@@ -29,11 +29,13 @@
 
 COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
 ifeq ($(COCOTB_CONFIG_CMD),)
-    $(error 'Cocotb is not installed (cocotb-config not found). \
+    $(error 'cocotb-config not found! Cocotb not installed (or installed but not on the PATH) \
              Please refer to https://cocotb.readthedocs.io/en/latest/quickstart.html#installing-cocotb \
              for installation instructions.')
 endif
 
+"
```
Comment (cmarqu) [inline makefiles/Makefile.inc:37]: Accident?

File: makefiles/Makefile.inc [hunk 2]
```diff
@@ -29,11 +29,13 @@
 
 COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
 ifeq ($(COCOTB_CONFIG_CMD),)
-    $(error 'Cocotb is not installed (cocotb-config not found). \
+    $(error 'cocotb-config not found! Cocotb not installed (or installed but not on the PATH) \
```
Comment (cmarqu) [inline makefiles/Makefile.inc:32]:
```suggestion
    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not in PATH) \
```

File: makefiles/Makefile.inc (hunk 1 above)
Comment (themperek) [reply makefiles/Makefile.inc:37]: yes

File: makefiles/Makefile.inc [hunk 3]
```diff
@@ -29,11 +29,13 @@
 
 COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
 ifeq ($(COCOTB_CONFIG_CMD),)
-    $(error 'Cocotb is not installed (cocotb-config not found). \
+    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not in PATH) \
              Please refer to https://cocotb.readthedocs.io/en/latest/quickstart.html#installing-cocotb \
              for installation instructions.')
 endif
 
+"
```
Comment (themperek) [inline makefiles/Makefile.inc:37]:
```suggestion
```

File: makefiles/Makefile.inc [hunk 4]
```diff
@@ -29,7 +29,7 @@
 
 COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
 ifeq ($(COCOTB_CONFIG_CMD),)
-    $(error 'Cocotb is not installed (cocotb-config not found). \
+    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not in PATH) \
```
Comment (eric-wieser) [inline makefiles/Makefile.inc:32]:
```suggestion
    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not on the PATH). \
```

File: makefiles/Makefile.sim [hunk 5]
```diff
@@ -29,7 +29,7 @@
 
 COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
 ifeq ($(COCOTB_CONFIG_CMD),)
-    $(error 'Cocotb is not installed (cocotb-config not found). \
+    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not in PATH) \
```
Comment (eric-wieser) [inline makefiles/Makefile.sim:32]:
```suggestion
    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not on the PATH). \
```

File: makefiles/Makefile.sim [hunk 6]
```diff
@@ -29,7 +29,7 @@
```
Comment (eric-wieser) [inline makefiles/Makefile.sim:29]: Given this file contains none of its previous content, I think we can easily replace this header with a SPDX one

**Pull request merged**